.github/workflows/update-changelog.yml

```
@@ -0,0 +1,124 @@
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Get PR details
+        id: pr
+        run: |
+          pr_number=${{ github.event.issue.number }}
+          pr_data=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls/$pr_number")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)